### PR TITLE
Added tests for checkbox

### LINKED ISSUE #936: Add tests for Checkbox

### PR DESCRIPTION
**Pull request by prasidhprabudhan**
Fixes #936

### CODE REVIEW

**Comment by prasidhprabudhan** [comment]
@ajmaln _a
Please review.

File: tests/Checkbox.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Checkbox } from "../lib/components";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("Checkbox", () => {
+  it("should render without error", () => {
+    const { getByText } = render(<Checkbox label="Checkbox" />);
+    expect(getByText("Checkbox")).toBeInTheDocument();
+  });
```
Comment (ajmaln) [inline tests/Checkbox.test.js:10]: Let's do `getByRole("checkbox")` or `getByLabelText("Checkbox")` instead of `getByText` here.

**Review by ajmaln — COMMENTED** [review]
@prasidhprabudhan _a A minor change

**Comment by prasidhprabudhan** [comment]
@ajmaln _a
Please review

**Review by ajmaln — APPROVED** [review]

**Pull request merged**